data/src/main/java/io/gnosis/data/repositories/TokenRepository.kt

```diff
@@ -13,27 +16,10 @@ class TokenRepository(
     suspend fun loadBalanceOf(safe: Solidity.Address): CoinBalances {
         val response = gatewayApi.loadBalances(safe.asEthereumAddressChecksumString())
         return CoinBalances(response.fiatTotal, response.items.map {
-
-            if (it.tokenInfo.address == null || it.tokenInfo.address == ZERO_ADDRESS)
-                Balance(
-                    ETH_TOKEN_INFO,
-                    it.balance,
-                    it.fiatBalance
-                )
+            if (it.tokenInfo.address == ZERO_ADDRESS)
+                it.copy(tokenInfo = it.tokenInfo.copy(logoUri = "local::ethereum"))
```

Comment:
@jpalvarezl We should probably have this (logoUri for ETH transfers) in the client gateway?